paymentsheet/src/main/java/com/stripe/android/paymentsheet/analytics/EventReporter.kt

```diff
@@ -189,4 +191,14 @@ internal interface EventReporter {
     enum class CardBrandChoiceEventSource {
         Edit, Add
     }
+
+    companion object {
+        fun ErrorReporter.report(errorEvent: ErrorReporter.ErrorEvent, stripeException: StripeException) {
```

Comment:
This was a mistake on my end. I meant `ErrorReporter`, not `EventReporter` 🙈

Reply:
ah gotcha. Updated. I replaced the existing `report` function with this one. If we need a different `report` function/need to overload it, we can add that when it's used